lib/large_object_store.rb

```diff
@@ -53,6 +55,11 @@ def read(key)
         return nil if slices.compact.size < pages
         slices.join("")
       end
+
+      if data.getbyte(0) == 0x78 && [0x01,0x9C,0xDA].include?(data.getbyte(1))
```

Comment:
is this how rails handles it ?
in theory this could fail if I want to store something already compressed ... but that should not happen ...